### PR TITLE
Adding browser and webview skip_broker config options to test app

### PR DESCRIPTION
**Pull request by melissaahn**
### Summary
This PR adds two testing config files to msalTestApp:
- msal_config_browser_skip_broker.json: same as msal_config_browser.json, except includes `"broker_redirect_uri_registered": false`
- msal_config_webview_skip_broker.json: same as msal_config_webview.json, except includes `"broker_redirect_uri_registered": false`

These two config files may help for when we want to test MSAL alone without having to remove the broker app from our test device, or if we would want to create different scenarios for testing CCT.

### CODE REVIEW

**Review by somalaya — APPROVED** [review]

**Comment by shahzaibj** [comment]
I thought we already had at least one skip browser config file in MSAL Test App.

This one: https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/testapps/testapp/src/main/res/raw/msal_config_instance_aware_common_skip_broker.json

File: testapps/testapp/src/main/res/raw/msal_config_browser_skip_broker.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "broker_redirect_uri_registered": false,
+  "authorities" : [
+    {
+      "type": "AAD",
+      "audience": {
+        "type": "AzureADMultipleOrgs"
+      }
+    }
+  ]
+}
```
Comment (shahzaibj) [inline testapps/testapp/src/main/res/raw/msal_config_browser_skip_broker.json]: nit: new line ending

File: testapps/testapp/src/main/res/raw/msal_config_webview_skip_broker.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "webauthn_capable" : true,
+  "authorities" : [
+    {
+      "type": "AAD",
+      "audience": {
+        "type": "AzureADMultipleOrgs"
+      }
+    }
+  ]
+}
```
Comment (shahzaibj) [inline testapps/testapp/src/main/res/raw/msal_config_webview_skip_broker.json]: nit: new line ending

**Review by shahzaibj — APPROVED** [review]

File: testapps/testapp/src/main/res/raw/msal_config_browser_skip_broker.json (hunk 1 above)
Comment (melissaahn) [reply testapps/testapp/src/main/res/raw/msal_config_browser_skip_broker.json]: 5a476deddaee5fce71c88cc641e7c4eae3b0a427

File: testapps/testapp/src/main/res/raw/msal_config_webview_skip_broker.json (hunk 2 above)
Comment (melissaahn) [reply testapps/testapp/src/main/res/raw/msal_config_webview_skip_broker.json]: 5a476deddaee5fce71c88cc641e7c4eae3b0a427

**Comment by melissaahn** [comment]
> I thought we already had at least one skip browser config file in MSAL Test App.
> 
> This one: https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/testapps/testapp/src/main/res/raw/msal_config_instance_aware_common_skip_broker.json

Yep, and this one is based on the msal_config_instance_aware.json file. I'm creating the other two based on their respective non-skip-broker files.

**Pull request merged**